libraries/AP_ToshibaCAN/AP_ToshibaCAN.cpp

```diff
@@ -539,4 +552,53 @@ void AP_ToshibaCAN::send_esc_telemetry_mavlink(uint8_t mav_chan)
     }
 }
 
+//
+// accessors for latest data from esc
+//
+// return latest rpm
+uint16_t AP_ToshibaCAN::get_rpm(uint8_t esc_id) const
+{
+    if (esc_id > TOSHIBACAN_MAX_NUM_ESCS) {
+        return 0;
+    }
+    return _telemetry[esc_id].rpm;
+}
+
+// return voltage in volts
+float AP_ToshibaCAN::get_voltage(uint8_t esc_id) const
+{
+    if (esc_id > TOSHIBACAN_MAX_NUM_ESCS) {
+        return 0;
+    }
+    return _telemetry[esc_id].voltage_cv / 100.0f;
```

Comment:
micro opt, but `* 1e-2f`

Reply:
Txs.  I've removed this function for now because it's unused.